src/main/java/de/filefighter/rest/configuration/CorsConfig.java

```diff
@@ -0,0 +1,28 @@
+package de.filefighter.rest.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.web.cors.CorsConfiguration;
+import org.springframework.web.cors.UrlBasedCorsConfigurationSource;
+import org.springframework.web.filter.CorsFilter;
+
+import java.util.ArrayList;
+
+@Configuration
+public class CorsConfig {
+
+    // Cors again. For local testing only.
+    @Bean
+    @Profile("dev")
```

Comment:
we also need this on prod for development,

otherwise we cant use https (if we use corsanywhere)

Reply:
i dont get it. Why are we needing cors again.

Reply:
if we want to develop locally we want to use the backend on the server. For this we either need cors or use corsanywhere. Because the ssl cert is weird we can't use corsanywhere with ssl.

Reply:
then we can deploy the container with dev -Spring Profile.

Reply:
then we have to change this in the ClientSetup